presidio-anonymizer/presidio_anonymizer/entities/anonymizer_config.py

```diff
@@ -0,0 +1,56 @@
+"""Handle the anonymizers data - anonymizer class and params."""
+import logging
+
+from presidio_anonymizer.anonymizers import Anonymizer
+from presidio_anonymizer.entities import InvalidParamException
+
+
+class AnonymizerConfig:
+    """Handle the anonymizers data - anonymizer class and params."""
```

Comment:
Missing docstring for anonymizer_name and params

Reply:
Added in the init